README.md

```diff
@@ -27,14 +27,14 @@ use postgis_diesel::types::*;
 #[derive(Insertable)]
 #[diesel(table_name = geometry_samples)]
 struct NewGeometrySample {
-    point: Point,
+    poinP: Point,
```

Comment:
I believe that change was an unexpected one. Why name was changed?

Reply:
Definitely unexpected, I will fix it asap sorry